CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set(ddesc "An object-oriented component library supporting radiation")
 string(APPEND ddesc " transport applications.")
 project( Draco DESCRIPTION ${ddesc} VERSION 7.8 LANGUAGES CXX C)
 
+# Export compile commands for compiler tools
+set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
```

Comment:
What's this change to the CMake? Otherwise, this PR LGTM

Reply:
@attom @timmah FYI - this is useful for LLVM tools with negligible side effects.

Reply:
This directs CMake to emit a "compilation database",  a JSON file with all the compile command lines listed. Clang tools like clang-tidy, clang-query etc use it to get things like compile arguments, include paths, etc.